### PR TITLE
add fixed assets / settings / locations

### PR DESCRIPTION
**Pull request by Hassan-SoftMachine**

### CODE REVIEW

**Comment by RebeccaFrem02** [comment]
image


i guess there is something wrong with the Transaction log button, check it with Nora

File: src/pages/fa-locations/forms/LocationForm.js [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+import ResourceComboBox from 'src/components/Shared/ResourceComboBox'
+import { SystemRepository } from '../../../repositories/SystemRepository'
+import { ControlContext } from 'src/providers/ControlContext'
+import { GeneralLedgerRepository } from 'src/repositories/GeneralLedgerRepository'
+
+const validationSchema = yup.object({
+  reference: yup.string().required(),
+  name: yup.string().required(),
+  plantId: yup.string().required()
+})
```
[the remark this answers is not in the document]
Comment (HassanKourani) [reply src/pages/fa-locations/forms/LocationForm.js:23]: Dont you think its cleaner this way?

[the remark this answers is not in the document]
Comment (RebeccaFrem02) [reply src/pages/fa-locations/forms/LocationForm.js:23]: this is the convention that all the team is using, please follow it so there will be no confusion

**Comment by Hassan-SoftMachine** [comment]
> image
> 
> i guess there is something wrong with the Transaction log button, check it with Nora

old data do not work, new data work fine

**Review by RebeccaFrem02 — APPROVED** [review]

**Pull request merged**